js/schema/style.json

```diff
@@ -29,6 +29,15 @@
     "size": {
       "type": "string",
       "enum": ["small", "medium", "large"]
+    },
+    "fill": {
+      "type": "string",
+      "format": "color",
+      "maxLength": 256
+    },
+    "button-size": {
```

Comment:
Since you have a new `buttonSizeToPx` function to convert it, can we just use the same `size` element from the json schema? I don't think we need a separate `button-size` property. We just need to make sure the `sizeToPx` and `buttonToPx` produce values that scale proportionately together.